README.md

```diff
@@ -1,17 +1,30 @@
 # rentify
 a service for landlords deployed as a collection of microservices running in docker containers
 
-### languages
-Java 8/9
-
-
-### frameworks
-Spring Boot 2.0
-Spring Cloud
-Netflix 0SS
-https://projectreactor.io/
-
-### other
-OAUTH 
-JWT
+## Technology Stack
+1. Languages (JVM based)
+  - Java (minimum v9)
+  - Scala
+  - Kotlin
+  - Groovy
+2. API Specification
+  - [OPENAPI](https://openapis.org)
+  - [RAML] (https://raml.org)
+  - [Hydra] (www.hydra-cg.com)
+  - [JSON Hyper-Schema] (http://json-schema.org/latest/json-schema-hypermedia.html)
+3. Frameworks
+  4. Web
+    - [Spring Boot 2.0 (WebFlux)](https://projects.spring.io/spring-boot/)
+    - [Vert.x](https://vertx.io)
+    - [Lagom](https://www.lagomframework.com/)
+    - [Ratpack](https://ratpack.io/)
+  5. Others
+      - [Spring Cloud/Netflix 0SS] (http://projects.spring.io/spring-cloud/)
+      - [Keycloak](keycloak.org)
 
+## Decisions
+1. Languages: All languages from above
```

Comment:
Let's use micro services and implement each one in a different language, so that we can compare

Reply:
👍 should we use different languages/frameworks?
1. Spring(Boot) officially support Java, Groovy, Kotlin.
2. Vert.x officially support Java, Groovy, Kotlin, Scala (and Ceylon, Ruby, JS)
3. Lagom officially Scala and Java
4. Ratpack officially Java and Groovy

Reply:
Exactly